### ISSUE TITLE
Editor LinkList cache not regenerated

### ISSUE THREAD

**Issue by jessedobbelaere** [forkcms/forkcms#1400]
A client mentioned an issue on his site: edit a page, then select a word in ckeditor and press the anchor icon in ckeditor. The "create-link" modal should open but it didn't show up, only the overlay was active and blocking the screen from further interaction.

Found out that I cleared the cache a while ago using `sh remove_cache`.  Which means `/src/Frontend/Cache/Navigation/editor_link_list_nl.js` also gets removed. I worked for some time in Fork after that. Then created an anchor link in an existing page and the link window didn't open.

Did a fresh install of the latest fork, installed with example data and debug enabled. Go straight to a page and make an anchor from a word. Same issue. Because of the example data, you didn't add/delete a page yourself and the editor_link_list cache wasn't yet builded --> var linkList = null --> anchor modal crashes

I had to add a new page and delete it again, to trigger the editor_link_list.js cache file rebuilding. Shouldn't this be generated much sooner, whenever fork detects that the file doesn't exist or so? Now the link-modal panel opens and crashes because `linkList = null` and the grey overlay blocks the whole screen. 

---
**tl;dr:** When you `sh remove_cache`, the `/src/Frontend/Cache/Navigation/editor_link_list_nl.js` gets deleted but not regenerated when you start using Fork. When you want to create a link in ckeditor, the link-modal crashes because var LinkList=null. You have to add/delete a page to make the linklist regenerate. Till then, the anchor-modal is not usable. 
Video of the problem: http://d.pr/v/vnyA

**Comment by jeroendesloovere** [comment]
Closed since bugfix solved this.
